spec/services/sufia/admin_set_create_service_spec.rb

```diff
@@ -2,7 +2,8 @@
 
 RSpec.describe Sufia::AdminSetCreateService do
   let(:admin_set) { AdminSet.new(title: ['test']) }
-  let(:service) { described_class.new(admin_set, user) }
+  let(:workflow_name) { nil }
+  let(:service) { described_class.new(admin_set, user, workflow_name) }
```

Comment:
Checking on this…when you pass the nil workflow name I believe the tests will fail. Also, with the change in the admin set default name, I believe a spec in this file needs updating (consider referencing the constant instead of using a literal).